### ISSUE TITLE
Empty Grid fails accessibility check as it has element with role="rowgroup", that does not contain row elements

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#6278]
### Bug report
Empty Grid fails accessibility check as it has element with role="rowgroup", that does not contain row elements. For reference see the WAI-ARIA spec:
https://www.w3.org/TR/wai-aria/#rowgroup
```
Required Owned Elements: | row
```

Edit:

The same is also true for the `role="row"` elements, they also must contain child elements:

https://www.w3.org/TR/wai-aria/#row

### Reproduction of the problem
Run the following Dojo and inspect the DOM: https://dojo.telerik.com/@Dimitar-Goshev/AYiSaRow 
The `<tbody role="rowgroup">` does not contain any row elements.

### Expected/desired behavior
role="rowgroup" requires an owned element with role="row"

### Environment
* **Kendo UI version:** 2023.3.1114
* **Browser:** [all]
